scripts/check-protoc.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
I pulled this PR locally and ran `make` which complained about unavailability of protoc 

~~~
./scripts/check-protoc.sh: line 25: protoc: command not found
Protobuf compiler (protoc)  or later is missing on this system.
Install protoc  using the preferred method for your system.
Refer to https://developers.google.com/protocol-buffers/ if Protobuf  is not available in the system repositories.
Makefile:17: recipe for target 'check-protoc' failed
make: *** [check-protoc] Error 1
~~~

Did `sudo dnf install /usr/bin/protoc` and got the packages. But the above error didn't go away :(

~~~
[ppai@gd2-1 glusterd2]$ which protoc
/usr/bin/protoc
~~~